ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -630,6 +630,14 @@ StackView {
 
                 collectibleName: view.token.name
                 model: view.tokenOwnersModel || null
+                accounts: root.accounts
+                feeText: root.feeText
+                chainName: view.token.chainName
+                isFeeLoading: root.isFeeLoading
+                feeErrorText: root.feeErrorText
+                onOpened: {
```

Comment:
I think we don't need this signal to be triggered `onOpened` bc when we open the popup the idle state will be without the fees box available, just with it's placeholder, so requests will start when some changes happen in the form.